### PR TITLE
Update README.md with "multimodal data" messaging

### PR DESCRIPTION
**Pull request by nikolausWest**
### What
Update intro text, code snippet, and screenshot
Screenshot 2023-09-27 at 16 26 17

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/3498) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/3498)
- [Docs preview](https://rerun.io/preview/c2e882af8f07f470718dbb1f9400eb63a9eb2849/docs) 
- [Examples preview](https://rerun.io/preview/c2e882af8f07f470718dbb1f9400eb63a9eb2849/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

**Comment by emilk** [comment]
[Rendered](https://github.com/rerun-io/rerun/blob/niko-update-readme/README.md)

File: README.md [hunk 1]
```diff
@@ -12,24 +12,28 @@
...
+# rr.save("recording.rrd")  # Stream all logs to disk
+
+# Associate subsequent data with 42 on the “frame” timeline
+rr.set_time_sequence("frame", 42))
+
+# Log colored 3D points to the entity at `path/to/points`
+rr.log("path/to/points", rr.Points3D(positions, colors=colors)
 …
 ```
+![rerun-github-hero](https://github.com/rerun-io/rerun/assets/2624717/c4900538-fc3a-43b8-841a-8d226e7b5a2e)
```
Comment (emilk) [inline README.md:35]: Why no longer centered?

And what is up with that alt-text ("rerun-github-hero") ?

**Review by emilk — APPROVED** [review]
Very nice!

File: README.md [hunk 2]
```diff
@@ -12,24 +12,28 @@
...
-rr.log_rect("car", bbox)
+rr.connect()  # Connect to a remote viewer
+# rr.spawn()  # Spawn a child process with a viewer and connect
+# rr.save("recording.rrd")  # Stream all logs to disk
+
+# Associate subsequent data with 42 on the “frame” timeline
+rr.set_time_sequence("frame", 42))
+
+# Log colored 3D points to the entity at `path/to/points`
+rr.log("path/to/points", rr.Points3D(positions, colors=colors)
```
Comment (teh-cmc) [inline README.md:32]: OCD triggered

```suggestion
rr.log("path/to/points", rr.Points3D(positions, colors=colors))

**Comment by teh-cmc** [comment]
What about "About"? :zany_face: 
image

File: README.md (hunk 1 above)
Comment (nikolausWest) [reply README.md:35]: I just dragged it in and forgot to update the alt-text. I'll try centering it again

File: README.md [hunk 3]
```diff
@@ -12,24 +12,28 @@
   <a href="https://discord.gg/Gcm8BbTaAj">                                <img alt="Rerun Discord"  src="https://img.shields.io/discord/1062300748202921994?label=Rerun%20Discord"> </a>
 </h1>
 
-# Rerun: Visualization infrastructure for computer vision.
+# An SDK for building time aware visualizations of multimodal data
```
[the remark this answers is not in the document]
Comment (nikolausWest) [reply README.md:15]: Good points!

**Pull request merged**